<?xml version="1.0" encoding="UTF-8"?>
<!--
    Copyright 2020 Dmitry Sokolov <mr.dmitry.sokolov@gmail.com>

    Licensed under the Apache License, Version 2.0 (the "License");
    you may not use this file except in compliance with the License.
    You may obtain a copy of the License at

        http://www.apache.org/licenses/LICENSE-2.0

    Unless required by applicable law or agreed to in writing, software
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
  -->
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qLogcat</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="spacing">
    <number>8</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0">
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>8</number>
            </property>
            <property name="rightMargin">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>4</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="pidFilterLabel">
              <property name="text">
               <string>PID filter</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QLineEdit" name="pidFilterEdit"/>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="pidFilterInvertedFlag">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Inverted</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_8">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>8</number>
            </property>
            <property name="rightMargin">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>4</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="ppidFilterLabel">
              <property name="text">
               <string>PPID filter</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QLineEdit" name="ppidFilterEdit"/>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="ppidFilterInvertedFlag">
              <property name="text">
               <string>Inverted</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_7">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>8</number>
            </property>
            <property name="rightMargin">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
            <property name="verticalSpacing">
             <number>4</number>
            </property>
            <item row="0" column="0">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="nameFilterLabel">
              <property name="text">
               <string>Name filter</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="nameFilterInvertedFlag">
              <property name="text">
               <string>Inverted</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QLineEdit" name="nameFilterEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>8</number>
            </property>
            <property name="rightMargin">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
            <property name="verticalSpacing">
             <number>4</number>
            </property>
            <item row="2" column="0" colspan="2">
             <widget class="QLineEdit" name="priorityFilterEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="priorityFilterLabel">
              <property name="text">
               <string>Priority filter</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="priorityFilterInvertedFlag">
              <property name="text">
               <string>Inverted</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>8</number>
            </property>
            <property name="rightMargin">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
            <property name="verticalSpacing">
             <number>4</number>
            </property>
            <item row="2" column="0" colspan="2">
             <widget class="QLineEdit" name="tagFilterEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="tagFilterLabel">
              <property name="text">
               <string>Tag filter</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="tagFilterInvertedFlag">
              <property name="text">
               <string>Inverted</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="autoscrollFlag">
           <property name="text">
            <string>Auto scroll</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="autosizeBtn">
           <property name="text">
            <string>Fit columns</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="filterBtn">
           <property name="text">
            <string>Apply filter</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableView">
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
